modules/core/src/main/scala/lucuma/odb/api/model/ValidatedInput.scala

```diff
@@ -15,19 +18,33 @@ object ValidatedInput {
     as.flattenOption match {
       case List(a) => a
       case Nil     => InputError.missingInput(name).invalidNec[A]
-      case _       => InputError.fromMessage(s"Multiple '$name' definitions are not permitted").invalidNec[A]
+      case _       =>
+        InputError.fromMessage(s"Multiple '$name' definitions are not permitted").invalidNec[A]
     }
 
-  def optionEither[A, B](nameA: String, nameB: String, a: Option[ValidatedInput[A]], b: Option[ValidatedInput[B]]): ValidatedInput[Option[Either[A, B]]] =
+  def optionEither[A, B](
+    nameA: String,
+    nameB: String,
+    a:     Option[ValidatedInput[A]],
+    b:     Option[ValidatedInput[B]]
+  ): ValidatedInput[Option[Either[A, B]]] =
     List(a.map(_.map(_.asLeft[B])), b.map(_.map(_.asRight[A]))).flattenOption match {
       case List(c) => c.map(_.some)
       case Nil     => Option.empty[Either[A, B]].validNec[InputError]
-      case _       => InputError.fromMessage(s"Either $nameA or $nameB are permitted but not both").invalidNec
+      case _       =>
+        InputError.fromMessage(s"Either $nameA or $nameB are permitted but not both").invalidNec
     }
 
   def nonEmptyString(name: String, s: String): ValidatedInput[NonEmptyString] =
     NonEmptyString
       .from(s)
       .leftMap(err => InputError.fromMessage(s"'$name' may not be empty: $err"))
       .toValidatedNec
+
+  def closedInterval[T](name: String, value: T, low: T, high: T)(implicit
+    v:                        Validate[T, Interval.Closed[low.type, high.type]]
```

Comment:
Scalafmt?

Reply:
Sigh. Yeah. I think the scalafmt config we use for Explore is too aggressive in aligning things and I just copied it to use here. I'll try some things to make it less awful for this case. Speaking of which, if we aren't using a common scalafmt in this project, should I add `.scalafmt.conf` to the `.gitignore` file?

Reply:
Yeah, that's probably a good idea.  I'm not convinced it makes the formatting better.

Reply:
I have tweaked the scalafmt config file I'm using to format the above better. I will push a commit with that change. I will use scalafmt on files I create, but try to manually format changes to existing files.

I don't think anyone would argue that automated formatting is better than artisanally crafted formatting. The advantages in my mind are:

1. Consistency in a project with multiple authors - esp. as team size gets bigger.
2. I don't need to spend time or brain cycles on formatting. I just type and let scalafmt handle it. No need to go back and fix vertical alignment if I change something, etc.

Oh, and 3), formatting is no longer a code review issue. 😆 

You can probably never get exactly what you want with scalafmt, but tweaking the file can get close. For the most egregious cases, I just do `// format: off`

Reply:
I concede point 1 but I think I prefer inconsistency over what you get with scalafmt.  Points 2 and 3 are often claimed but seem to be the opposite of reality -- I suppose because of people like me who whine about what scalafmt produces.  :-)